src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -135,6 +157,13 @@ export default class NewDataset extends Vue {
     T: null,
     C: null
   };
+  strides: { [dimension: string]: number | string | null } = {
+    XY: null,
```

Comment:
Why do you need to support 'string' type when it used to support only number?

Reply:
The input for the stride is a `v-text-field` and its v-model uses a string instead of a number.

Reply:
I think you can specify the input type of the v-text-field https://vuetifyjs.com/en/api/v-text-field/#props-type